app/views/users/new.html.haml

```diff
@@ -0,0 +1,25 @@
+%section.register.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      =bootstrap_form_for @user, class: "form-horizontal" do |f|
```

Comment:
The proper way to do horizontal forms is now `layout: horizontal`. You can read that in the current gem docs.